### PR TITLE
Add possibility to insert iptables rule

### PR DESCRIPTION
**Pull request by polinaag**

### CODE REVIEW

File: rrmngmnt/firewall.py [hunk 1]
```diff
@@ -144,6 +144,25 @@ def add_rule(self, dest, target, protocol='all', ports=None):
...
+            dest (dict): 'address' key and value containing destination host or
+               list of destination hosts
+            target (str): Target rule to apply
+            protocol (str): affected network protocol, Default is 'all'
+            ports (list): list of ports to configure
+
+        Returns:
+            bool: False if inserting new rule failed, True if it succeeded
+        """
+        return self.edit_chain(
```
Comment (myakove) [inline rrmngmnt/firewall.py]: Maybe better to set by rule number.
iptables -D FORWARD 1, for example, this one will be the first one.

Comment (cynepco3hahue) [reply rrmngmnt/firewall.py]: I do not sure, but I think it works only for delete.

Comment (myakove) [reply rrmngmnt/firewall.py]: iptables -I INPUT 1 -i lo -j ACCEPT
https://serverfault.com/questions/374993/how-do-you-prepend-an-iptables-rather-than-append

Comment (polinaag) [reply rrmngmnt/firewall.py]: I've added rule number. Please have a look.

File: rrmngmnt/firewall.py [hunk 2]
```diff
@@ -89,16 +89,21 @@ def edit_chain(
...
-            '--jump', target.upper(), '--protocol', protocol
+            self.firewall_service, action, chain_name
         ]
 
+        if rule_num:
+            cmd.extend([rule_num])
+
+        dest = ",".join(dest['address'])
+        cmd.extend([address_type, dest,
+            '--jump', target.upper(), '--protocol', protocol])
```
Comment (myakove) [inline rrmngmnt/firewall.py]: (
....
)

File: rrmngmnt/firewall.py [hunk 3]
```diff
@@ -144,6 +149,26 @@ def add_rule(self, dest, target, protocol='all', ports=None):
...
+        """
+        Insert new firewall rule to a specific chain
+
+        Args:
+            dest (dict): 'address' key and value containing destination host or
+               list of destination hosts
+            target (str): Target rule to apply
+            protocol (str): affected network protocol, Default is 'all'
+            ports (list): list of ports to configure
+            rule_num (str): rule number to insert
```
Comment (myakove) [inline rrmngmnt/firewall.py]: More info on rule number please, explain that this will put the rule in x line

File: rrmngmnt/firewall.py (hunk 2 above)
Comment (polinaag) [reply rrmngmnt/firewall.py]: done

File: rrmngmnt/firewall.py [hunk 4]
```diff
@@ -89,16 +91,28 @@ def edit_chain(
...
+            self.firewall_service, action, chain_name
         ]
 
+        if rule_num:
+            cmd.extend([rule_num])
+
+        dest = ",".join(dest['address'])
+        cmd.extend(
+            [
+                address_type, dest,'--jump', target.upper(),
```
Comment (lukas-bednar) [inline rrmngmnt/firewall.py]: could you please put space after comma (`dest, `) to make pep8 happy ?

**Pull request merged**